packages/office-ui-fabric-react/src/components/Callout/examples/Callout.Directional.Example.tsx

```diff
@@ -31,8 +32,14 @@ const DIRECTION_OPTIONS = [
   { key: DirectionalHint.rightBottomEdge, text: 'Right Bottom Edge' }
 ];
 
+const getClassNames = classNamesFunction<{}, ICalloutExampleStyle>();
+const classNames = getClassNames(calloutExampleStyle, {});
+
 export class CalloutDirectionalExample extends React.Component<{}, ICalloutDirectionalExampleState> {
   private _menuButtonElement: HTMLElement | null;
+  // Use getId() to ensure that the callout title ID is unique on the page.
+  // (It's also okay to use a plain string without getId() and manually ensure its uniqueness.)
+  private _titleId: string = getId('callout-label');
```

Comment:
What's the reasoning behind adding this ID? I see it being attached to an element but not referenced anywhere else.

Reply:
I saw this in other file and was confused but removed it!